io/jvm-native/src/main/scala/fs2/io/file/Path.scala

```diff
@@ -88,4 +88,5 @@ object Path extends PathCompanionApi {
   private def apply(path: JPath): Path = new Path(path)
   def apply(path: String): Path = fromNioPath(FileSystems.getDefault.getPath(path))
   def fromNioPath(path: JPath): Path = Path(path)
+  def fromFsPath(fs: FileSystem, path: String): Path = fromNioPath(fs.getPath(path))
```

Comment:
`getPath` is a var-args, should this API match?
https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileSystem.html#getPath-java.lang.String-java.lang.String...-

Reply:
I guess I chose consistency with other `Path` constructors over API match here. In the end user can use other `Path` functions to get the same result.

Reply:
Good point! :)